src/FarmCraft.Core/Services/Messaging/Publisher/ServiceBusPublisher.cs

```diff
@@ -4,34 +4,37 @@
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json;
 
-namespace FarmCraft.Core.Services.Messaging
+namespace FarmCraft.Core.Services.Messaging.Publisher
 {
     /// <summary>
     /// A Message Publisher for interacting with Azure Service Bus
     /// </summary>
     public class ServiceBusPublisher : IMessagePublisher, IDisposable
     {
         private readonly ILogger _logger;
-        private static ServiceBusClient _client;
-        private static ServiceBusSender _sender;
+        private static ServiceBusSender? _sender;
```

Comment:
This can't be static if there will be more than one publisher. Test to make sure this works correctly if registering multiple instances as singletons, or maybe rethink this a bit.

Reply:
This should work a non-static sender. Thinking along the lines of:

1. Register the MessageBusService as a singleton.
2. Each actor is responsible for creating a publisher if it needs to publish messages to a queue / topic. The constructor gets the singleton, and creates however many publishers needed. Actors that publish should probably be long lived instead of using the "Handle with instance of" where they get killed off
3. The actor disposes of the publisher when shutting down